specs/LaunchingRegisteredProtocols.md

```diff
@@ -0,0 +1,318 @@
+# Background
+
+We are exposing an event that will be raised when an attempt to launch a protocol that is registered with the OS (external protocol) is made.
+When navigating to a URI, the URI scheme determines how to handle the URI.
+Some schemes like http, and https, are resolved by WebView2 and the navigation is handled in the WebView2.
+Other URI schemes may be registered externally to the WebView2 with the OS by other applications.
+Such schemes are external protocols and are handled by launching the registered application with the URI.
+
+The host will be given the option to cancel the external protocol launch with the `LaunchingExternalProtocol` event.

```

Comment:
All the existing CoreWebView2 events have their verb at the end. (overall event ends with -ed or -ing.)  Maybe "ExternalProtocolLaunching" ?

Reply:
We do have precedent for verb-first events: Application.LeavingBackground, CoreInputView.XYFocusTransferringFromPrimaryView, FocusManager.Getting/LosingFocus, ListView.ChoosingItemContainer, ScrollViewer.BringingIntoView. I think verb-first is okay, particularly for "-ing" events. We're allowed to override a rule to avoid an awkward name.

Reply:
Between
* `LaunchingExternalUriScheme`
* `ExternalUriSchemeLaunching`

Launching... sounds better. We'll use that.